docs/resources/dns_auth_zone.md

```diff
@@ -13,6 +13,14 @@ Manages an authoritative zone.
 ## Example Usage
 
 ```terraform
+resource "bloxone_keys_tsig" "test" {
+  name = "test-tsig."
+}
+
+resource "bloxone_dns_acl" "test" {
+  name = "test-acl"
+}
```

Comment:
Even though an empty ACL is possible, it is not useful. For our examples, it would be better to have an ACL with basic list, where the transfer_acl can follow the following order 

deny any
allow 192.168.1.0/24
deny ip 192.168.1.1
allow tsig test-tsig.

Reply:
Addressed

Reply:
I see that you have removed the nested acl altogether. Sorry if I was not clear in my comments. We do want the nested ACL as well, but not a nested ACL that is empty. So the whole resource should look something like this

```
resource "bloxone_dns_auth_zone" "example" {
  fqdn         = "domain.com."
  primary_type = "cloud"

  # Other optional fields
  comment = "Example of an Authoritative Zone"
  tags = {
    site = "Site A"
  }
  query_acl = [
    {
      access  = "deny"
      element = "ip"
      address = "192.168.1.1"
    },
    {
      element = "acl"
      acl = bloxone_dns_acl.example_acl.id
    },
    {
      access  = "allow"
      element = "tsig_key"
      tsig_key = {
        key = bloxone_keys_tsig.example_tsig.id
      }
    },
    {
      access  = "deny"
      element = "any"
    },
  ]
  update_acl = [
    {
      access  = "allow"
      element = "any"
    },
  ]
  transfer_acl = [
    {
      access  = "allow"
      element = "any"
    },
  ]

}

resource "bloxone_keys_tsig" "example_tsig" {
  name = "example_tsig.domain.com."
}

resource "bloxone_dns_acl" "example_acl" {
    name = "example_acl"
    elements = [
        {
        access  = "deny"
        element = "ip"
        address = "192.168.1.0/24"
        },
    ]
}
```